pinot-integration-tests/src/test/java/org/apache/pinot/compat/tests/CompatibilityOpsRunner.java

```diff
@@ -54,11 +59,11 @@ private boolean runOps() throws Exception {
   }
 
   public static void main(String[] args) throws Exception {
-    if (args.length != 2) {
+    if (args.length != 3) {
```

Comment:
Only two args needed.yaml file name and gen number

Reply:
Passing third parameter - testSuiteDir(Parent dir) will help for the part 2 of the issue. It will allow different filePaths under yaml files to be relative to the directory. Or do we want to look for alternative?

Reply:
If you pass the abs path of the yaml file, you can assume in the java code that the dir is one level above the abs path.

Reply:
done!